app/code/Magento/Cms/Model/BlockManagment.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Cms\Model;
+
+use Magento\Cms\Api\BlockManagementInterface;
+use Magento\Cms\Api\Data\BlockInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+/**
+ * @api
+ */
```

Comment:
`@api` annotation is generally not needed for the classes which implement api interface. 
It may be added only in cases when class has broader contract then interface.